File: RepoTransBench/target_projects/Java/Go/sockeqwe_fragmentargs/public_tests/protectedaccess_public_test.go
package public_tests

import (
	"testing"
)

func TestProtectedFieldAccessTestPublicVariant(t *testing.T) {
	dummy := &DummyProtectedObjectPublic{}
	dummy.SetValue("barBaz")
	if got := dummy.GetValue(); got != "barBaz" {
		t.Errorf("Expected \"barBaz\", got %q", got)
	}
}

type DummyProtectedObjectPublic struct {
	Field string
}

func (d *DummyProtectedObjectPublic) SetValue(val string) {
	d.Field = val
}
func (d *DummyProtectedObjectPublic) GetValue() string {
	return d.Field
}

File: RepoTransBench/target_projects/Java/C#/sockeqwe_fragmentargs/public_tests/ProtectedAccessPublicTest.cs
using Xunit;

namespace ProjectName.PublicTests.Processor
{
    public class ProtectedAccessPublicTest
    {
        [Fact]
        public void ProtectedFieldAccessTest_PublicVariant()
        {
            var dummy = new DummyProtectedObjectPublic();
            dummy.SetValue("barBaz");
            Assert.Equal("barBaz", dummy.GetValue());
        }

        public class DummyProtectedObjectPublic
        {
            protected string? field;
            public void SetValue(string val) => this.field = val;
            public string? GetValue() => this.field;
        }
    }
}